pkg/generate/apigwv2_test.go

```diff
@@ -226,4 +227,13 @@ func TestAPIGatewayV2_Route(t *testing.T) {
 	}
 `
 	assert.Equal(expCreateOutput, crd.GoCodeSetOutput(model.OpTypeCreate, "resp", "ko.Status", 1))
+
+	expRequiredStatusFieldsMissingFromReadOneInput := `
+	if r.ko.Status.RouteID == nil  {
+		return true
+	} else {
+		return false
+	}
+`
+	assert.Equal(expRequiredStatusFieldsMissingFromReadOneInput, fmt.Sprintf("\n%s\n", crd.GoCodeRequiredStatusFieldsMissingFromReadOneInput("r.ko", 1)))
```

Comment:
Instead of the above, a cleaner approach might be:

```go
got := crd.GoCodeRequiredStatusFieldsMissingFromReadOneInput("r.ko", 1)
assert.Equal(strings.Trim(expRequiredStatusFieldsMissingFromReadOneInput), got)
```